docs/generators/spring.md

```diff
@@ -83,6 +83,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
 |sourceFolder|source folder for generated code| |src/main/java|
+|sse|boolean, toggles to generate server send events using spring boot SseEmitter. only available, if 'reactive' option is active| |false|
```

Comment:
I think this would be more readable if it was `serverSideEvents` instead of `sse`

Reply:
"sse" is the short hand side for "Server-sent events", not server side events. I would choose that phrase. Ok?

Reply:
Sounds good to me 👍

Reply:
solved